src/main/java/reposense/report/RepoCloner.java

```diff
@@ -68,11 +68,18 @@ public RepoLocation getClonedRepoLocation() {
             String bareRepoPath = FileUtil.getBareRepoPath(configs[currentIndex]).toString();
             currentRepoDefaultBranch = GitBranch.getCurrentBranch(bareRepoPath);
         } catch (GitBranchException gbe) {
-            // GitBranch will throw this exception when repository is empty
             logger.log(Level.WARNING, String.format(MESSAGE_ERROR_GETTING_BRANCH,
```

Comment:
Why do we need this given you have added code for this already? Or did you forget to remove it?

Reply:
I didn't remove this exception handling as I think that there might be some other errors that may occur. For instance, GitBranchException is thrown because an IOException occurred, which probably happened if the directory given is not a Git repository (which shouldn't happen).